src/rastervision/common/tasks/train_model.py

```diff
@@ -81,6 +81,13 @@ def get_lr(epoch):
             callback = LearningRateScheduler(get_lr)
             callbacks.append(callback)
 
+        if self.options.epoch_decay:
```

Comment:
Instead of `decay` and `epoch_decay`, I think `lr_step_decay` and `lr_epoch_decay` would be clearer.